### ISSUE TITLE
Missing Alt Text - Homepage

### ISSUE THREAD

**Issue by c-deems** [alaoweb/2021.alaoweb.org#101]
Per the Pa11y Dashboard, an image on the conference homepage is missing alt text via the alt attribute.

Link to the Homepage in Pa11y Dashboard: http://pa11y.alaoweb.org/60f09064fae3030008f1eae8

Impacted line:
`<img class="serviceproject img" src="/img/sponsors/dfs_logo.jpg">`
